src/style.css

```
@@ -0,0 +1,36 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-image: url(images/star-active.svg);
+}
+
+.stars--0 .stars__star:nth-child(n) {
+  background-image: url(images/star.svg);
+}
+
+.stars--1 .stars__star:nth-child(n + 2) {
+  background-image: url(images/star.svg);
+}
```

combine all these styles into one

```suggestion
.stars--0 .stars__star:nth-child(n),
.stars--1 .stars__star:nth-child(n + 2) {
  background-image: url(images/star.svg);
}
```